pkg/apis/flows/v1alpha1/sequence_types.go

```diff
@@ -78,6 +78,17 @@ type SequenceSpec struct {
 	Reply *duckv1.Destination `json:"reply,omitempty"`
 }
 
+type SequenceStep struct {
+	// Subscriber receiving the step event
+	Subscriber duckv1.Destination `json:",inline"`
+
+	// Delivery is the delivery specification for events to the subscriber
+	// This includes things like retries, DLQ, etc.
+	// Needed for Roundtripping v1alpha1 <-> v1beta1.
+	// +optional
+	Delivery *eventingduckv1alpha1.DeliverySpec `json:"delivery,omitempty"`
```

Comment:
How about we make this v1beta1 right off the bat?

Reply:
sure.